src/components/EmojiPicker/EmojiPicker.js

```diff
@@ -134,6 +134,9 @@ const EmojiPicker = forwardRef((props, ref) => {
             });
         });
         return () => {
+            if (!emojiPopoverDimensionListener) {
```

Comment:
Why this change?

Reply:
One of the tests was throwing a lot of errors because of the listener was null, was impossible to fix the unite tests from ReportActionsList without this
@MonilBhavsar

Reply:
Thanks for clarifying 👍